examples/wafv2-and-or-rules/main.tf

```diff
@@ -0,0 +1,106 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+#####
+# Web Application Firewall configuration
+#####
+module "waf" {
+  source = "../.."
+
+  name_prefix = "test-waf-setup"
+  alb_arn     = aws_lb.test.arn
```

Comment:
this should be a working examples that deploys correct, this alb doesn't exist so don't create attachment to an ALB

Reply:
Removed ALB